packages/daemon/src/identity/PeerUserIdentity.js

```diff
@@ -91,4 +97,8 @@ export default class PeerUserIdentity extends Identity {
   get avatar(): ?string {
     return this._profile.avatar
   }
+
+  updateProfile(profile: PeerUserProfile = {}) {
```

Comment:
Where do we land on setters vs update methods?

Reply:
Good question!
I prefer methods when there are possible side-effects so there can be an implicit understanding by the caller that it involves computation, so I'll change this to a setter and we can continue using methods for other cases maybe?